src/components/App/App.css

```diff
@@ -1,70 +1,142 @@
 main {
+  background-color: black;
   height: 100vh;
   width: 100vw;
 }
 
-h1 {
-  color: #ad9976;
-}
-
 .window {
   display: grid;
-  height: 90vh;
+  height: 85%;
```

Comment:
Why the conversion here? It's my understanding that vh is more static across devices than percentages.

Reply:
You right